servicesContracts.py

```diff
@@ -1,10 +1,28 @@
 from typing import Any
-
-from artifacts import *
 from servicesContracts.requestEthereum_service import RequestEthereumService
 
+"""
+  getServiceFromAddress return the service of a coresponding currency contract address
+  :param  address:
+        The address of the currency contract
+"""
 def getServiceFromAddress(address: str) -> Any:
-    pass
+    if not address:
+        return None
+    if isThisArtifact(Artifacts.requestEthereumArtifact, address):
+        return RequestEthereumService()
```

Comment:
Seems strange to me that the return is not based upon the address at all, is this correct?

Reply:
Yes, it's correct based on JS implementation. https://github.com/RequestNetwork/requestNetwork.js/blob/e8bdb9a8a762fceba8bea91a91cafe06a9f92f1a/src/servicesContracts.ts#L15

Reply:
Ok, looks good then.